websites/D/DashNet/presence.ts

```diff
@@ -29,10 +29,9 @@ presence.on("UpdateData", () => {
 	if (document.location.pathname.includes("/cookieclicker/")) {
 		const cookies = document
 			.querySelector("#cookies")
-			.textContent.replace(
-				document.querySelector("#cookies div").textContent,
-				""
-			);
+			.innerHTML.replace("<br>", " ")
+			.split("<")[0];
```

Comment:
You should never use innerHTML because of these hacky things it makes you do. Here's a cleaner solution
```ts
const cookies = document.querySelector("#cookies").innerText.split("\n")
presenceData.state = cookies.pop()
presenceData.details = cookies.join(" ")
```
This should work for when the cookie number is too large to fit in 1 line too